ospfd/ospf_vty.c

```diff
@@ -1292,19 +1239,56 @@ DEFUN (no_ospf_area_vlink,
   return ospf_vl_set (ospf, &vl_config);
 }
 
+DEFUN (ospf_area_vlink_intervals,
+       ospf_area_vlink_intervals_cmd,
+       "area <A.B.C.D|(0-4294967295)> virtual-link A.B.C.D {hello-interval (1-65535)|retransmit-interval (1-65535)|transmit-delay (1-65535)|dead-interval (1-65535)}",
```

Comment:
Personally I'd prefer to break this long line into smaller ones like we do in `show_ip_bgp_cmd`. But we have a few other very long commands in ospfd, maybe we can create a separate PR for that later.